packages/integration-oauth/src/providers/google.ts

```diff
@@ -53,7 +53,7 @@ export const google = <_Auth extends Auth>(auth: _Auth, config: Config) => {
 				client_id: config.clientId,
 				redirect_uri: redirectUri ?? config.redirectUri,
 				scope: scope(
-					["https://www.googleapis.com/auth/userinfo.profile"],
+					["https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/userinfo.email"],
```

Comment:
I'm against adding the email scope by default; it should be opt-in. I think `GoogleUser.email` should be made optional instead.

Reply:
how would you suggest we make it optional?

Reply:
Already fixed this in #682:

```ts
type GoogleUser = {
  email?: string
}
```

Reply:
Nice, take note that the `email_verified` field should also be optional.

Reply:
bump